tests/test_e2e.py

```diff
@@ -29,13 +31,13 @@ def test_e2e_run(cli_runner: CliRunner, data_dir: Path, tmp_path: Path) -> None:
             str(data_dir / "rules" / "example_user_rules.yml"),
             "run",
             str(data_dir / "input" / "tiff"),
+            "--output-dir",
             str(tmp_path),
         ],
     )
 
     assert result.exit_code == 0
-    output_file = tmp_path / "REDACTED_test_image.tif"
-    assert output_file.exists()
+    output_file = tmp_path / "Redacted_2023-05-12_12-12-53/REDACTED_test_image.tif"
```

Comment:
These need to be separate components.

Reply:
fixed in a0c2bf04704b088fb3ba1cfdeb17ff930b8983bf